README.md

```diff
@@ -64,7 +64,7 @@ To terminate the running tool, call `http://localhost:42800/send/kill`.
 
 This is yet another small tool to enhance the power of AHK. Some more links, you might find interesting:
 
-- **[AHK2PremiereCEP](https://github.com/sebinside/AHK2PremiereCEP)**, another utility tool from me. It helps you connect authotkey with the Adobe Premiere CEP scripting environment. Very helpful tool for video production.
+- **[AHK2PremiereCEP](https://github.com/sebinside/AHK2PremiereCEP)**, another utility tool from me which helps you connect Authotkey with the Adobe Premiere CEP scripting environment. A very helpful tool for video production.
```

Comment:
"AutoHotkey" is the official capitalization of the word "autohotkey" ;)

source: https://www.autohotkey.com/

Reply:
Ok, my bad. Fixed missed AutoHotkey instances with the wrong capitalization and fixed all instances of Authotkey.